tasks/main.yml

```diff
@@ -196,6 +204,48 @@
     (__mssql_server_packages not in ansible_facts.packages) or
     (mssql_upgrade | bool)
 
+- name: Configure to run as a confined application with SELinux
+  when: __mssql_confined_supported | bool
+  block:
+    - name: Ensure that the system is in SELinux enforcing state
+      when: mssql_manage_selinux | bool
+      include_role:
+        name: fedora.linux_system_roles.selinux
+      vars:
+        selinux_state: "{{
+          'enforcing' if mssql_run_selinux_confined | bool
+          else 'permissive' }}"
+
+    - name: Get the current mode of SELinux
```

Comment:
Is it necessary to check that the selinux role set the correct mode?

Reply:
Selinux role is run when `mssql_manage_selinux: true`, if `mssql_manage_selinux: false` the role gets the current mode and then verifies if the mode matches the chosen scenario confined vs non-confined